app/javascript/components/requests/Request.jsx

```diff
@@ -90,53 +189,69 @@ class Request extends React.Component {
             </StyledModal>
           </li>
           <li value={id} onClick={() => this.deleteRequest(id)}>Delete</li>
-          <li>Block user</li>
+          { 
+            this.state.isBlocking ?
+              <li value={id} onClick={this.unblockUser}>Unblock user</li> : 
```

Comment:
What's the purpose of the `value={id}` attribute? I think `value` is for the counter in ordered lists: https://html.com/attributes/li-value/

Reply:
Removed because it didn't seem necessary for functionality!